### PR TITLE
Use the updated RTC in AraRoot

### PR DESCRIPTION
**Pull request by clark2668**
The AraRoot RayTraceCorrelator got major speed upgrades. This adapts AraProc to those updates.

### CODE REVIEW

File: araproc/analysis/standard_reco.py [hunk 1]
```diff
@@ -258,108 +270,78 @@ def do_standard_reco(self, wavepacket):
...
+        ####### VPol Pulser ########
+        ############################
+
         # check the cal pulser in V
         pulser_map_v = self.rtc_wrapper.correlators["nearby"].GetInterferometricMap(
-            self.pairs_v,
-            self.__corr_functions_v,
-            0
-        )
+            self.pairs_v, self.__corr_functions_v, self.__arrival_delays_v, 0,)
```
Comment (marcomuzio) [inline araproc/analysis/standard_reco.py:283]: Idk if this is a comment or curiosity, but is the hanging comma important?

**Comment by marcomuzio** [comment]
Looks good to me! I think you can merge.

File: araproc/analysis/standard_reco.py (hunk 1 above)
Comment (clark2668) [reply araproc/analysis/standard_reco.py:283]: It's the latest recommendation from e.g. ruff and flake8: https://docs.astral.sh/ruff/rules/missing-trailing-comma/

Comment (marcomuzio) [reply araproc/analysis/standard_reco.py:283]: Ohh very interesting! Never thought about that. Thanks!

**Comment by clark2668** [comment]
Thank you!

**Pull request merged**